src/storage/heap_file.c

```diff
@@ -25308,7 +25308,7 @@ heap_get_visible_version_internal (THREAD_ENTRY * thread_p, HEAP_GET_CONTEXT * c
 static int
 heap_update_set_prev_version (THREAD_ENTRY * thread_p, const HFID * hfid, const OID * oid,
 			      PGBUF_WATCHER * home_pg_watcher, PGBUF_WATCHER * fwd_pg_watcher,
-			      LOG_LSA * prev_version_lsa)
+			      const LOG_LSA * prev_version_lsa)
```

Comment:
Build fails.
The signature of the function must be changed everywhere.
Also the signature of or_mvcc_set_log_lsa_to_record must also be changed everywhere to:
```
int or_mvcc_set_log_lsa_to_record (RECDES * record, const LOG_LSA * lsa)
```
because the function does not change the data `lsa` points to.

Reply:
You're correct, I should've checked. :joy:

Reply:
Corrected.